lib/rototiller/task/params/command.rb

```diff
@@ -160,6 +159,11 @@ def run
         @result
       end
 
+      def stop
+        return true if [@switches, @options, @arguments].any?{ |collection| collection.stop? }
```

Comment:
wrong question?  i believe this has to stop if any env_var says to stop
so... `|| @env_vars.any? { |collection| collection.stop }`

unless you change it to stop? as in my suggestion above and maybe you answer here